Package.swift

```diff
@@ -49,15 +54,17 @@ let package = Package(
 		.target(
```

Comment:
We want to use Swift settings in SargonUniFFI target as well but we cannot due to https://github.com/mozilla/uniffi-rs/issues/2046